### DISCUSSION TITLE
Fixed size for the columnChart even if only one chart value is provided

### DISCUSSION THREAD

**Discussion by Aatif-Thinkitive** [patrykandpatrick/vico#378] [Questions] [answered]
i am getting values from API but if there is only one value the column Size gets bigger which I don't want to happen
![image](https://github.com/patrykandpatrick/vico/assets/108254342/ae16204f-eea6-4889-a040-6a2eede368d2)

I want it to be of this size even if there is only one column
![image](https://github.com/patrykandpatrick/vico/assets/108254342/c287526e-9281-414a-ac59-b439c7994326)

**Comment by patrickmichalik** [comment, marked as answer]
Hello! You can use `AutoScaleUp.None` to turn off this behavior. Please see the `autoScaleUp` parameter of the `Chart` composable function or the `autoScaleUp` field of `BaseChartView`.

**Reply by Aatif-Thinkitive** [reply]
Thank you, It worked!.

**Reply by patrickmichalik** [reply]
You’re welcome!

**Reply by vanikad** [reply]
Hello, 
Is this api removed in 2.0.0-beta.3? Cannot find it and not able to get fixed width for column chart.
Can anyone shed some light?

**Reply by Gowsky** [reply]
Hello, @vanikad. This functionality is still available, but it's now accessed via a different API: `rememberVicoZoomState`. This provides more flexibility.

You should adjust `initialZoom` (and possibly `minZoom` and `maxZoom`). See [`Zoom.Companion`](https://www.patrykandpatrick.com/vico/api/latest/core/com.patrykandpatrick.vico.core.cartesian/-zoom/-companion/index.html).

**Reply by marcosalis** [reply]
Hi @Gowsky , I managed to find this discussion after some proper fighting to get this working with Vico 2.
It's all good now, but don't you think that if `zoomEnabled = false` the other zoom options should be ignored?
Thank you!

**Reply by Gowsky** [reply]
> don't you think that if `zoomEnabled = false` the other zoom options should be ignored?

`zoomEnabled = false` means that zooming by gesture is off. Even if this is the case, you may still need to customize the chart’s scaling, which is why other options remain functional.

**Reply by marcosalis** [reply]
Hi @Gowsky , thank you for your reply. It's all on me, I misinterpreted the purpose of that option.
It works exactly as needed 👌
